template.yaml

```diff
@@ -0,0 +1,368 @@
+          ToPort: !Sub "{{resolve:secretsmanager:${RDSConnectionSecretArn}:SecretString:port}}"
+          DestinationSecurityGroupId: !Ref RDSClientSGID
+
+  # ECS Resources
+  HapiFhirCluster:
+    Type: AWS::ECS::Cluster
+    Properties:
+      ClusterName: !Sub "${TestPrefix}${Env}hapi-fhir-cluster"
+      ClusterSettings:
+        - Name: containerInsights
+          Value: enabled
+
+  HapiFhirTaskDefinition:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      Family: !Sub "${TestPrefix}${Env}hapi-fhir-task"
+      NetworkMode: awsvpc
+      RequiresCompatibilities:
+        - FARGATE
+      Cpu: 2048
+      Memory: 4096
+      ExecutionRoleArn: !GetAtt ECSTaskExecutionRole.Arn
+      TaskRoleArn: !GetAtt ECSTaskRole.Arn
+      RuntimePlatform:
+        OperatingSystemFamily: LINUX
+        CpuArchitecture: ARM64
+      ContainerDefinitions:
+        - Name: !Sub "${TestPrefix}${Env}hapi-fhir-container"
+          Image: !Ref EcrImageUri
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref HapiFhirLogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: ecs
+          PortMappings:
+            - ContainerPort: 8080
+              HostPort: 8080
+              Protocol: tcp
+          Environment:
+            - Name: SPRING_DATASOURCE_URL
+              Value: !Join
+                - ""
+                - - "jdbc:postgresql://"
+                  - !Sub "{{resolve:secretsmanager:${RDSConnectionSecretArn}:SecretString:host}}"
+                  - ":"
+                  - !Sub "{{resolve:secretsmanager:${RDSConnectionSecretArn}:SecretString:port}}"
+                  - "/"
+                  - !Ref HapiFhirDBName
+            - Name: SPRING_DATASOURCE_USERNAME
+              Value: !Sub "{{resolve:secretsmanager:${RDSCredentialsSecretArn}:SecretString:username}}"
+            - Name: SPRING_DATASOURCE_PASSWORD
+              Value: !Sub "{{resolve:secretsmanager:${RDSCredentialsSecretArn}:SecretString:password}}"
+            - Name: POSTGRES_DB
+              Value: !Ref HapiFhirDBName
+            - Name: POSTGRES_USER
+              Value: !Sub "{{resolve:secretsmanager:${HapiFhirCredentialsSecretArn}:SecretString:username}}"
+            - Name: POSTGRES_PASSWORD
+              Value: !Sub "{{resolve:secretsmanager:${HapiFhirCredentialsSecretArn}:SecretString:password}}"
+
+  HapiFhirALB:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties:
+      Name: !Sub "${TestPrefix}${Env}-hapi-fhir-alb"
+      Scheme: internal
+      LoadBalancerAttributes:
+        - Key: idle_timeout.timeout_seconds
+          Value: "60"
+      SecurityGroups:
+        - !Ref ALBSecurityGroup
+      Subnets: !Ref PrivateSubnets
+
+  ALBListener:
+    Type: AWS::ElasticLoadBalancingV2::Listener
+    Properties:
+      LoadBalancerArn: !Ref HapiFhirALB
+      Port: 80
```

Comment:
Is the intention to map from port 80 to port 8080?   Or should this listen on port 8080 to?